### PR TITLE
Adding InternalAction with similar signature to OnEntryFrom

### PR DESCRIPTION
**Pull request by HenningNT**
As requested in #76 I have added an overload with the same signature as OnEntryFrom.

### CODE REVIEW

File: Stateless/StateConfiguration.cs [hunk 1]
```diff
@@ -48,41 +48,61 @@ public StateConfiguration Permit(TTrigger trigger, TState destinationState)
                 return InternalPermit(trigger, destinationState, string.Empty);
             }
             /// <summary>
-            /// 
+            /// Add an internal transition to the state machine. An internal action does not cause the Exit and Entry actions to be triggered, and does not change the state of the stae machine
```
Comment (tlk) [inline Stateless/StateConfiguration.cs]: small typo: "stae machine" -> "state machine" (replicated below)

**Comment by tlk** [comment]
Hi @HenningNT This is my first time using the new github review function - please bear over with me...

**Comment by HenningNT** [comment]
No worries, I'm still trying to figure out how github works :D

**Comment by HenningNT** [comment]
Hi!
I think I did it!
This time I used Visual Studio to do the pulling/fetching/whatnot :-)
I think it's ready!

**Comment by tlk** [comment]
Looks good to me -- @nblumhardt any thoughts?

**Comment by nblumhardt** [comment]
Looks good to me - just one thing, it needs to go to `dev` now rather than `master`. Will change branch and merge :+1:

**Pull request merged**